app/models/assignment.rb

```diff
@@ -457,7 +457,87 @@ def add_csv_group(row)
     end
     collision_error
   end
+  def add_csv_groups(file)
```

Comment:
Perceived complexity for add_csv_groups is too high. [9/7]<br>Use empty lines between defs.